form.yml

```diff
@@ -7,7 +7,20 @@ form:
   - imagefile
   - custom_num_cores
 attributes:
-  imagefile: "scipy-notebook_lab-4.0.7.sif"
+  imagefile:
+    display: true
+    widget: "select"
+    label: "Jupyter Lab Container"
+    options:
+      - ["scipy-notebook", "scipy-notebook.sif"]
+      - ["datascience-notebook", "datascience-notebook.def"]
```

Comment:
Should this be `datascience-notebook.sif`?

Reply:
Yes, thank you! That would make the app not work...